### ISSUE TITLE
Simplify using SceneLoader.load_scene_async_with_cb

### ISSUE THREAD

**Issue by workhorsy** [ImmersiveRPG/GodotAsyncLoader#9]
Using SceneLoader.load_scene_async_with_cb is confusing.

**Comment by workhorsy** [comment]
For example:

```GDScript
# Instance scene asynchronously and send to callback
var target = get_tree().get_current_scene()
var scene_file := "res://examples/Animals/Puma.tscn"
var pos := Vector3(0, 1, 0)
var is_pos_global := false
SceneLoader.load_scene_async_with_cb(target, scene_file, pos, is_pos_global, funcref(self, "on_animal_loaded"), {})

func on_animal_loaded(path : String, instance : Node, pos : Vector3, is_pos_global : bool, data : Dictionary) -> void:
	var target = get_tree().get_current_scene()
	instance.transform.origin = pos
	target.add_child(instance)
```

**Comment by workhorsy** [comment]
A better option:

**Comment by workhorsy** [comment]
Fixed in #14
